core/event.c

```diff
@@ -1382,7 +1382,7 @@ int event_queue_add_timer(int eq, int *id, int sec) {
 	*id = timer_id;
 	timer_id++;
 
-	EV_SET(&kev, *id, EVFILT_TIMER, EV_ADD, 0, sec * 1000, 0);
+	EV_SET(&kev, *id, EVFILT_TIMER, EV_ADD, 0, sec * 1000 + int(nsec / 1000000), 0);
```

Comment:
The unit of the 6th parameter isn't immediately clear... Could it somehow be made clearer?